app/views/articles/_form.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= form_with(model: article) do |form| %>
```

Comment:
@sansuaza how the `form_with model:` works?

Reply:
once the model to which the form belongs is indicated (model: article), we can start to build the form with "form", indicating which attribute of the article belongs to the field